docs/docs/glossary/webpack.md

```diff
@@ -0,0 +1,31 @@
+---
+title: webpack
+disableTableOfContents: true
+---
+
+Learn what webpack is, how it works, and how Gatsby uses it to accelerate web site development.
+
+## What is webpack?
+
+[webpack](/docs/glossary#webpack) is a <q>static module bundler,</q> or software that collects chunks or modules of JavaScript and compiles them into one or more optimized bundles. webpack is one of the core software packages that underpins Gatsby.
+
+webpack works by creating a _dependency graph_. In other words, webpack determines which modules depend on other modules to make your site work. A module is simply a chunk of code that encapsulates some functionality. It may be as small as a single JavaScript function, or it may be an entire library such as [React](/docs/glossary#react).
```

Comment:
```suggestion
webpack works by creating a _dependency graph_. In other words, webpack determines which modules depend on other modules to make your site work. A module is a chunk of code that encapsulates some functionality. It may be as small as a single JavaScript function, or it may be an entire library such as [React](/docs/glossary#react).
```